aviary/examples/multi_mission/run_multimission_example_large_single_aisle.py

```diff
@@ -272,32 +272,32 @@ def createN2(fileref, prob):
         createN2(__file__, super_prob)
 
     super_prob.run()
-    printoutputs = [
-        (Mission.Design.GROSS_MASS, 'lbm'),
-        (Aircraft.Design.EMPTY_MASS, 'lbm'),
-        (Aircraft.Wing.SWEEP, 'deg'),
-        (Aircraft.LandingGear.MAIN_GEAR_MASS, 'lbm'),
-        (Aircraft.LandingGear.NOSE_GEAR_MASS, 'lbm'),
-        (Aircraft.Design.LANDING_TO_TAKEOFF_MASS_RATIO, 'unitless'),
-        (Aircraft.Furnishings.MASS, 'lbm'),
-        (Aircraft.CrewPayload.PASSENGER_SERVICE_MASS, 'lbm'),
-        (Mission.Summary.GROSS_MASS, 'lbm'),
-        (Mission.Summary.FUEL_BURNED, 'lbm'),
-        (Aircraft.CrewPayload.PASSENGER_MASS, 'lbm'),
-        (Aircraft.CrewPayload.PASSENGER_PAYLOAD_MASS, 'lbm'),
-        (Aircraft.CrewPayload.CARGO_MASS, 'lbm'),
-        (Aircraft.CrewPayload.TOTAL_PAYLOAD_MASS, 'lbm')]
-    super_prob.print_vars(vars=printoutputs)
-
-    plotvars = [('altitude', 'ft'),
-                ('mass', 'lbm'),
-                ('drag', 'lbf'),
-                ('distance', 'nmi'),
-                ('throttle', 'unitless'),
-                ('mach', 'unitless')]
-    super_prob.create_timeseries_plots(plotvars=plotvars, show=False)
-
-    plt.show()
+    # printoutputs = [
+    #     (Mission.Design.GROSS_MASS, 'lbm'),
+    #     (Aircraft.Design.EMPTY_MASS, 'lbm'),
+    #     (Aircraft.Wing.SWEEP, 'deg'),
+    #     (Aircraft.LandingGear.MAIN_GEAR_MASS, 'lbm'),
+    #     (Aircraft.LandingGear.NOSE_GEAR_MASS, 'lbm'),
+    #     (Aircraft.Design.LANDING_TO_TAKEOFF_MASS_RATIO, 'unitless'),
+    #     (Aircraft.Furnishings.MASS, 'lbm'),
+    #     (Aircraft.CrewPayload.PASSENGER_SERVICE_MASS, 'lbm'),
+    #     (Mission.Summary.GROSS_MASS, 'lbm'),
+    #     (Mission.Summary.FUEL_BURNED, 'lbm'),
+    #     (Aircraft.CrewPayload.PASSENGER_MASS, 'lbm'),
```

Comment:
Since this is an example, I would just put this whole block of code under an `if show_plots:` conditional. Then, just set show_plots to false above. A user can easily set it to true to see the plots.